js/models.js

```diff
@@ -150,7 +166,7 @@ class User {
       data: { user: { username, password } },
     });
 
-    let { user } = response.data;
+    let { user } = response.data; // does this need to be enclosed in brackets?
```

Comment:
Yes, this is object destructuring: https://www.javascripttutorial.net/es6/javascript-object-destructuring/